python/ray/_private/runtime_env/context.py

```diff
@@ -46,14 +49,18 @@ def exec_worker(self, passthrough_args: List[str], language: Language):
             executable = f'"{self.py_executable}"'  # Path may contain spaces
         elif language == Language.PYTHON:
             executable = f"exec {self.py_executable}"
+        elif language == Language.JAVA:
+            executable = f"java"
+            ray_jars = os.path.join(get_ray_jars_dir(), "*")
```

Comment:
Do we need "*.jar", it seems safer.

Reply:
If it's not a jar, it should work as well. For example, the following directory should work:
![Uploading image.png…]()